realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProcessor.java

```diff
@@ -124,7 +124,7 @@
         "io.realm.annotations.RealmModule",
         "io.realm.annotations.Required"
 })
-@SupportedOptions(value = {"debug"})
+@SupportedOptions(value = {"realm.suppressWarnings"})
```

Comment:
I think this value could use the constant defined in `RealmProcessor` if that is made package visibility.